### PR TITLE
Fix nits in the change runtime containerd page

### PR DESCRIPTION
**Pull request by tengqm**
There are two commits in the PR. The first one fixes a link. The second one applies some line wrapping changes and fixes some trivial nits.

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md [hunk 1]
```diff
@@ -4,20 +4,28 @@ weight: 8
 content_type: task 
 ---
 
-This task outlines the steps needed to update your container runtime to containerd from Docker. It is applicable for cluster operators running Kubernetes 1.23 or earlier. Also  this covers an example scenario for migrating from dockershim to containerd and alternative container runtimes can be picked from this [page](https://kubernetes.io/docs/setup/production-environment/container-runtimes/).
+This task outlines the steps needed to update your container runtime to containerd from Docker. It
+is applicable for cluster operators running Kubernetes 1.23 or earlier. Also  this covers an
```
Comment (reylejano) [inline content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md:8]: nit: extra space
```suggestion
is applicable for cluster operators running Kubernetes 1.23 or earlier. Also this covers an
```

Comment (divya-mohan0209) [inline content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md:8]:
```suggestion
is applicable for cluster operators running Kubernetes 1.23 or earlier. Also, an
```

File: content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md [hunk 2]
```diff
@@ -4,20 +4,28 @@ weight: 8
...
+example scenario for migrating from dockershim to containerd and alternative container runtimes
```
Comment (divya-mohan0209) [inline content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md:9]:
```suggestion
example scenario covering the migration from dockershim to containerd and alternative container runtimes
```

File: content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md [hunk 3]
```diff
@@ -4,20 +4,28 @@ weight: 8
...
+can be picked from this [page](/docs/setup/production-environment/container-runtimes/).
```
Comment (divya-mohan0209) [inline content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md:10]:
```suggestion
can be viewed [here](/docs/setup/production-environment/container-runtimes/).
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
I've reworded the initial paragraph so that the trail of though isn't lost midway. Open to suggestions if we can phrase it better though.

**Comment by sftim** [comment]
@divya-mohan0209 would you be happy to see these changes merge as-is? We could open a good-first-issue follow up that describes the extra tidying.

**Comment by divya-mohan0209** [comment]
@sftim : Absolutely! Not a problem at all :)

**Pull request merged**